server/@types/viewModels/index.d.ts

```diff
@@ -173,30 +173,33 @@ declare module 'viewModels' {
     updatedAt: Date
   }
 
+  export interface WorkInterestJob {
```

Comment:
Exporting this as a type was more convenient (for the comparator function and its tests) than defining it inline, which we did previously. It means the comparator can refer to its type 👍